auth-api/config.py

```diff
@@ -132,6 +132,12 @@ class _Config(object):  # pylint: disable=too-few-public-methods
     # url for the front end app
     WEB_APP_URL = os.getenv('WEB_APP_URL')
 
+    try:
+        MAX_NUMBER_OF_ORGS = int(os.getenv('MAX_NUMBER_OF_ORGS'))
```

Comment:
You can simplify this line using default value. int(os.getenv('MAX_NUMBER_OF_ORGS', 3))